pages/api/graphql-rest.page.ts

```diff
@@ -900,14 +897,11 @@ class MpdxRestApi extends RESTDataSource {
   //
 
   async googleAccounts() {
-    const { data }: { data: GoogleAccountsResponse[] } = await this.get(
-      'user/google_accounts',
-      {
-        sort: 'created_at',
-        include: 'contact_groups',
-      },
-    );
-    return GoogleAccounts(data);
+    const response = await this.get('user/google_accounts', {
```

Comment:
`GoogleAccountsResponse` doesn't seem to be used anywhere. Instead of removing it, it would be nice to update it to match the response from the REST API for the sake of documentation.